_layouts/home.html

```diff
@@ -13,12 +13,99 @@ <h2 id="countdown">&nbsp;</h2>
     </p>
   </div>
 
-  <div class="flex row-xs-center ">
-    <div class="btn-group-lg">
-      <a class="btn btn-lg btn-default btn-next btn-xs-full" href="{{ '/host/' | prepend: site.baseurl }}">Become a host&nbsp;&nbsp;<i class="fa fa-play" aria-hidden="true"></i></a>
-      <a class="btn btn-lg btn-default btn-next btn-xs-full" href="{{ '/participant/' | prepend: site.baseurl }}">Attend an event&nbsp;&nbsp;<i class="fa fa-play" aria-hidden="true"></i></a>
-    </div>
-  </div>
+  <!-- Add this css File in head tag-->
```

Comment:
this change removes the "Become a host" and "Attend an event" buttons. We will need those as you can see in issue #2.